steal.lic

```diff
@@ -52,7 +52,7 @@ class Steal
 
     settings = get_settings
     setup(settings)
-
+    start_script('performance', ['noclean']) unless settings.hide_to_steal
```

Comment:
I think we probably want to move starting performance down to below the checks that might cause the script to exit quickly. So, perhaps after line 73?

Reply:
Yeah, good catch.  There's also buffs in there, which I think means spellcasters with stealing buffs will start playing, try to buff, stop playing, buff, then begin stealing.  73 looks like good to me?

@Sarvatt got any thoughts?

Reply:
bput handles stopping playing and the script will start again between buffs so it wouldn't hurt anything, but I moved it anyway just to be extra safe :D